style.css

```diff
@@ -0,0 +1,135 @@
+body {
+  display: flex;
+  flex-direction: column;
+  align-content: space-around;
+  max-width: 1024px;
+  margin: 20px auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 0 50px;
+}
+h1 {
+  font-size: 2.5em;
+  font-weight: 800;
+}
+h2 {
+  letter-spacing: 0.5px;
+}
+.gradient {
+  background: linear-gradient(to right, rgb(143, 201, 224), rgb(255, 255, 255));
+}
+.header-name {
+  margin: 50px;
+}
+header li {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 10px;
+  margin-left: -40px;
+  list-style-type: none;
+  font-size: 1em;
+  font-weight: 700;
+  letter-spacing: 0.3px;
+}
+.contacts {
+  display: flex;
+  flex-direction: row;
+}
+.header-image {
+  margin: auto;
+  box-shadow: 3px 3px 3px rgb(128, 128, 128);
+}
+hr {
+  background-color: rgb(183, 215, 252);
+  height: 1px;
+  box-shadow: 0.5px 0.5px 0.5px rgba(0, 117, 250, 0.493);
+}
+.about-top {
+  margin: 0 100px 25px;
+  font-size: 1.2em;
+}
+/* .about-top h2 {
+  color: rgb(255, 255, 255);
+} */
+.about-last {
+  display: flex;
+  flex-direction: row;
+  margin: 0 100px 30px;
+  font-size: 1.2em;
+}
+.left {
+  padding: 20px 100px 0 0;
+  flex: 3;
+}
+.right {
+  padding: 20px 0;
+  flex: 2;
+}
+a {
+  color: rgb(108, 151, 189);
+}
+@media (min-width: 610px) and (max-width: 768px) {
+  header {
+    flex-direction: column;
+  }
+  .header-name {
+    margin: 0 auto;
+  }
+  .header-image {
+    margin-bottom: 20px;
+  }
+  .about-top {
+    margin-bottom: 0;
+  }
+}
+@media (max-width: 609px) {
```

Comment:
In the mobile version your contacts are not aligned at the center (see screenshot below):

![image](https://user-images.githubusercontent.com/45519864/103462668-d8c10300-4d37-11eb-8223-b4e60f39bd34.png)

Reply:
Done!